README.md

```diff
@@ -1,2 +1,40 @@
 # ringling-cli
 Command-line interface for the Ringling model management service
+
+## Installing ringling-cli
+Ringling-cli requires Python 3.8 or higher, and interfaces with [Ringling](https://github.com/msoe-dise-project/ringling)
+
+## Getting ringling-cli
+Download Ringling-cli by cloning the git repository:
+
+```bash
+$ git clone https://github.com/msoe-dise-project/ringling-cli
+```
+
+## Option 1. Install in Virtual Environment
+If you would prefer to use a virtual environment instead of installing system wide, you can:
+
+#### Linux:
+```bash
+$ cd ringling-cli
+$ python3 -m venv venv
+$ source venv/Scripts/activate
```

Comment:
If you're on Linux, MacOS, or similar, it's `source venv/bin/activate`

Reply:
When I install on Windows it doesn't create a venv/bin folder. Would it make sense to create separate Windows and Linux/MacOS instructions? Or just assume that the end user will figure it out?

Reply:
I would say leave it be for now